src/app/pages/NFTInstanceDashboardPage/InstanceDetailsCard.tsx

```diff
@@ -72,7 +72,7 @@ export const InstanceDetailsCard: FC<InstanceDetailsCardProps> = ({
                 </dd>
               </>
             )}
-            {nft?.num_transfers && (
+            {typeof nft?.num_transfers === 'number' && !isNaN(nft?.num_transfers) && (
```

Comment:
maybe we should show a row only for positive numbers for now? nft instance endpoint may return 0 even if there are transfer events in a tab below.